python/whylogs/api/fugue/profiler.py

```diff
@@ -0,0 +1,101 @@
+from datetime import datetime
+from functools import reduce
+from typing import Any, Dict, Iterable, List, Optional
+
+import pandas as pd
+from fugue import FugueWorkflow, PartitionSpec, Schema, transform, transformer
+
+import whylogs as why
+from whylogs.core.view.column_profile_view import ColumnProfileView
+from whylogs.core.view.dataset_profile_view import DatasetProfileView
+
+COL_NAME_FIELD = "__whylogs_fugue_col_name"
+COL_PROFILE_FIELD = "__whylogs_fugue_col_profile"
+COL_PROFILE_SCHEMA = Schema([(COL_NAME_FIELD, str), (COL_PROFILE_FIELD, bytes)])
+DF_PROFILE_FIELD = "__whylogs_df_profile_view"
+
+
+class _Profiler:
```

Comment:
Same as comment below on the name, but FugueDatasetPofiler would be more descriptive

Reply:
underscore prefixed names within a integration's folder are ok to be somewhat generic, its not part of whylogs.core where I would want more differentiation.